infra-templates/rbd/0/rancher-compose.yml

```diff
@@ -0,0 +1,20 @@
+.catalog:
+  name: "Rancher RBD"
+  version: "0.1.0"
```

Comment:
I wonder if we should match the version to the image version. it makes it easier to distinguish. Sorry, thought about it when reviewing the other PR.